dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -115,6 +117,17 @@ def model_install_cmds(output_file=None) -> str:
       return f"python install.py --continue_on_fail {pipe_file_cmd}"
     return f"python install.py models {model_name} {pipe_file_cmd}"
 
+  install_torch_xla2_dependency = (
+      (
+          # TODO(piz): torch_xla2 only support nightly test at this time.
+          "pip install torch_xla[pallas] -f https://storage.googleapis.com/jax-releases/jax_nightly_releases.html -f https://storage.googleapis.com/jax-releases/jaxlib_nightly_releases.html",
+          "cd ~/xla/experimental/torch_xla2/",
+          "pip install -e .",
```

Comment:
Should this be `.[tpu]` / `torch_xla2[tpu]`?

Reply:
I should mention this PR: https://github.com/pytorch/xla/pull/7126/. I am trying not to reinstall libtpu to an version that doesn't match with the torch_xla[pallas].
`pip install -e .` will not install jax/jaxlib. Instead we use the jax installation from torch_xla[pallas].

Reply:
Update to use .[tpu] after isolated the torch_xla and torch_xla2.